src/platform/packages/shared/kbn-esql-validation-autocomplete/src/shared/user_defined_columns.ts

```diff
@@ -137,6 +137,27 @@ export function collectUserDefinedColumns(
         addUserDefinedColumnFromExpression(node, queryString, userDefinedColumns, fields);
       }
     })
+    .on('visitForkCommand', () => {
```

Comment:
This is conceptually wrong, these are not user defined columns. 🤔 

I would keep the autocomplete as it was and create an extra function `systemDefinedColumns` or something like that and use them to validation. 

It makes it more clear that way I think